README.md

```diff
@@ -91,6 +91,73 @@ export INITIAL_CLONE_EMAIL="email@user.com"
 export INITIAL_CLONE_API_TOKEN="token"
 ```
 
+## Initial configuration
+
+In order to run fastlane.ci for the first time, the `https://github.com/your-name/your-ci-config` needs to be populated with at least two files.
+
+- `users.json`
+
+```json
+[
+    {
+      "id": "ee75eb27-9246-43c1-af5a-a8d33f8a963f",
+      "email": "your-name@gmail.com",
+      "password_hash": "some password hash that needs to be created",
+      "provider_credentials": [
+        {
+          "email": "minuscorp@gmail.com",
```

Comment:
best email

Reply:
Whoooops

Reply:
Hahaha, it's ok if you don't mind your email being in public. Mine is all over the place.